rr_platform/msg/chassis_state.msg

```diff
@@ -1,5 +1,5 @@
 # Message for defining the state of the car
 Header header           # header for when the message is published
-bool estop              # whether the estop is on or not
+bool estop_on           # whether the estop is turned on
```

Comment:
I'll change the comment, I don't know how, if at all, i should change the variable name to be more semantically meaningful